libs/services/designer-client-services/src/lib/standard/manifests/compose.ts

```diff
@@ -0,0 +1,45 @@
+import { coreBadge } from '../../badges';
+import { OutputSecureDataMode, SettingScope } from '@microsoft/utils-logic-apps';
+import type { OperationManifest } from '@microsoft/utils-logic-apps';
+
+export const composeManifest = {
```

Comment:
This file should be in src\lib\consumption/manifests since standard already has compose manifest and the operation manifest service should add it explicitly to the base value which is common between standard and consumption